lib/generator.js

```diff
@@ -229,8 +229,10 @@ class Generator {
   async generateFromString(asyncapiString, parserOptions = {}) {
     if (!asyncapiString || typeof asyncapiString !== 'string') throw new Error('Parameter "asyncapiString" must be a non-empty string.');
 
+    /** @type {String} AsyncAPI string to use as a source. */
     this.originalAsyncAPI = asyncapiString;
 
+    /** @type {Object} Parsed AsyncAPI schema. */
```

Comment:
```suggestion
    /** @type {AsyncAPIDocument} Parsed AsyncAPI schema. {@link https://github.com/asyncapi/parser-js/blob/master/API.md#module_@asyncapi/parser+AsyncAPIDocument|More explanation of object structure}. */
```

we can be more specific here and even link to docs

Reply:
Yes, I added the link and the description. Also fixed one typo.